### ISSUE TITLE
State dashboard: spacing

### ISSUE THREAD

**Issue by m-sullivan7** [cagov/covid19#5480]
In adding source data links to the [state dashboard](https://covid19.ca.gov/state-dashboard/) and [health equity](https://covid19.ca.gov/equity/), we'd like to even out the spacing for the source data link for hospitalized patients and ICU beds. I attempted to manipulate the `mt-#` and `mb-#` classes, but couldn't figure it out by the time I hit the limit of _Michael's safety guardrails against breaking code on pages_. I suspect this is pretty easy to fix (and probably related to the charts themselves), just beyond my powers!

State dashboard:
![image](https://user-images.githubusercontent.com/69169190/136997235-ea93ca12-a908-4743-afd4-7781c6e0e176.png)

Health equity:
![image](https://user-images.githubusercontent.com/69169190/137000712-c653c0ab-721a-4a96-84cc-a9c4da7e4db1.png)

Initial assignment to @mediajunkie just for today so as not to bother engineers on Tech Debt Tuesday with what is definitely a minor thing that doesn't need working on today.

**Comment by m-sullivan7** [comment]
It looks good to me! It's worth confirming with @mediajunkie before publishing.

**Comment by jbum** [comment]
State-dashboard is now fixed.  Moving on to the equity spacing issue...

@m-sullivan7 I've added a 'pb-1' qualifier here:

```
<p class="small-text mb-4 pb-1"><a href="https://data.chhs.ca.gov/dataset/covid-19-time-series-metrics-by-county-and-state">Cases relative to percentage of population and case rate per 100K by race and ethnicity group source data</a></p>
```

Which adds a little extra spacing on the bottom of the line, and helps even it out.  I've updated staging only.  If you like it, go ahead and push it. Otherwise, try a pb-2 (I think it looks fine with the pb-1, but I can barely see the outline of the box below it on my current monitor).
